src/scripts/components/foundations/Button.js

```diff
@@ -40,16 +69,38 @@ export const StyleAnchor = css`
   }};
 `
 
+export const ButtonStyle = css`
+  box-shadow: ${props => props.theme.colors.button.shadow};
+  cursor: ${props => (props.disabled ? 'initial' : 'pointer')};
+  font-size: ${props => (props.anchor ? null : props.theme.fonts.button)};
+  font-weight: ${props =>
+    props.anchor
+      ? props.theme.fonts.weight.regular
+      : props.theme.fonts.weight.bold};
+  margin: ${props => props.margin};
+  opacity: ${props => (props.disabled ? 0.5 : null)};
+  pointer-events: ${props => (props.disabled ? 'none' : null)};
+  text-transform: ${props => (props.anchor ? null : 'uppercase')};
+  user-select: ${props => (props.anchor ? null : 'none')};
+  text-rendering: geometricPrecision;
+`
```

Comment:
Can we move these to another file? Is this a good PR to refactor this component to be more generic? Fine if not 😄

Reply:
@bent0b0x let's create an issue for all those changes? 
What I think about: 
- Button(colored button)
- TextButton(actually Button)

And share(`export`) its styles.

Reply:
I think we should definitely create an issue to discuss further. 

Not sure about that distinction, though, and my initial thought is that we should export any styles from components. I believe there are other ways that we can share styles as needed, via constants and such. Component files should only be components, and should not have any other responsibilities like maintaining styles for other components (unless a component is a direct extension of said component). 

I also feel like if we need multiple components to share many of the same styles then perhaps we have not written our abstractions correctly, and we can refactor a bit

Reply:
https://github.com/Paratii-Video/paratii-portal/issues/737